packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -86,7 +86,9 @@ def format(self, record) -> str:
 # log_level=%{WORD:log_level} \| log_timestamp=%{TIMESTAMP_ISO8601:log_timestamp} \| log_source=%{DATA:log_source} \| log_msg=%{GREEDYDATA:log_msg}
 
 
-def config_all_loggers(*, log_format_local_dev_enabled: bool) -> None:
+def config_all_loggers(
+    *, log_format_local_dev_enabled: bool, logger_filter_mapping: dict[str, list[str]]
```

Comment:
Document your intent using code ... even if you do not do a hard validation here

e.g. (see notes below as well)

```python

LoggerName: TypeAlias = str
KeywordStr: TypeAlias = str


def foo(..., exclude_keywords_log_filter_args: dict[LoggerName, set[KeywordStr]] )
   ...
```


Let's talk about this offline.